### PR TITLE
Issue/#16

### PR DESCRIPTION
**Pull request by smnv-as**
All done!
It is first step in refactoring. I don't really think that I do all correctly. Check and give me feedback, please.

### CODE REVIEW

**Comment by smnv-as** [comment]
I made two more files.

**Comment by smnv-as** [comment]
Sorry!

File: flake8_class_attributes_order/node_type_weights.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
+import warnings
+
+from typing import List, Mapping, Dict
+from typing_extensions import Final
+
+
+class ClassNodeTypeWeights:
```
Comment (Melevir) [inline flake8_class_attributes_order/node_type_weights.py]: Lets remove this class and make its's content simple global vars.

**Review by Melevir — CHANGES_REQUESTED** [review]

**Comment by smnv-as** [comment]
Remove the class.

File: flake8_class_attributes_order/node_type_weights.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
+import warnings
+
+from typing import List, Mapping, Dict
+from typing_extensions import Final
+
+
+non_strict_node_type_weights: Final[Mapping[str, int]] = {
```
Comment (Melevir) [inline flake8_class_attributes_order/node_type_weights.py]: This shoud be UPPER_CASE now

File: flake8_class_attributes_order/node_type_weights.py [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    'static_method': 22,
+    'private_static_method': 22,
+    'class_method': 24,
+    'private_class_method': 24,
+    'method': 26,
+    'magic_method': 27,
+    'private_method': 27,
+}
+
+strict_node_type_weights: Final[Mapping[str, int]] = {
```
Comment (Melevir) [inline flake8_class_attributes_order/node_type_weights.py]: so as this

File: flake8_class_attributes_order/node_type_weights.py [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+    'static_method': 22,
+    'private_static_method': 23,
+    'class_method': 24,
+    'private_class_method': 25,
+    'method': 26,
+    'magic_method': 27,
+    'private_method': 28,
+}
+
+fixed_node_type_weights: Final[Dict[str, int]] = {
```
Comment (Melevir) [inline flake8_class_attributes_order/node_type_weights.py]: and this and other global vars inside this module

**Review by Melevir — CHANGES_REQUESTED** [review]
We're almost done :)

File: flake8_class_attributes_order/node_type_weights.py (hunk 4 above)
Comment (smnv-as) [reply flake8_class_attributes_order/node_type_weights.py]: All done!

**Comment by Melevir** [comment]
This one was huge. Thanks a lot!

**Pull request merged**